src/mediarithmics/plugins/common/BasePlugin.ts

```diff
@@ -27,21 +27,29 @@ import {
 
 import {Index, Option, flatMap} from '../../utils';
 import {normalizeArray} from '../../utils/Normalizer';
-import {DataResponse} from "../../";
+import {
+  asNativeDataProperty,
```

Comment:
Group `XXXProperty` / `asXXXProperty` together (we already have them imported above from `PluginPropertyInterface`).